doc/sphinx/source/notes_7_0.rst

```diff
@@ -95,47 +99,49 @@ CPU drivers
   an error flag, which the CPU driver then uses to set the NDRange
   event's state to failed.
 
-* Support for using LLVMSPIRVLib for SPIR-V input translation,
-  instead of llvm-spirv binary.
-
 * With "distro" builds, the kernel-lib variants now include
-  "generic" CPU variant as a fallback.
+  the "generic" CPU variant as a fallback.
 
 * OpenCL C printf() now correctly supports all OpenCL vector types.
 
-* Added option to disable support of subnormal floats.
+* Added option to disable the support of subnormal floats.
 
-* Added option to avoid division-by-zero exceptions using a LLVM pass
+* Added option to avoid division-by-zero exceptions using an LLVM pass
```

Comment:
I think "a LLVM pass" is the correct one.

Reply:
Nope!